cohttp-eio/src/client.ml

```diff
@@ -72,15 +72,16 @@ let response buf_read =
 
 let call ?meth ?version ?(headers = Http.Header.init ()) ?(body = Body.Empty)
     ~conn host resource_path =
+  let host_hdr = "Host" in
```

Comment:
Why this extra constant? It is not going to change ever, what is the advantage of allocating the extra variable?

Reply:
It is used in a few other places below. 

>what is the advantage of allocating the extra variable?

Interesting, is there an extra allocation when doing "let x = "Host" vs "Host"?

Reply:
Not sure to be honest. I assumed so unless the compiler unlined it away

Reply:
Done. I have removed the var declaration.